examples/interactive_segmentation/android/app/src/main/java/com/mediapipe/example/interactivesegmentation/InteractiveSegmentationHelper.kt

```diff
@@ -54,11 +54,14 @@ class InteractiveSegmentationHelper(
             val optionsBuilder =
                 InteractiveSegmenter.InteractiveSegmenterOptions.builder()
                     .setBaseOptions(baseOptions)
+                    .setOutputCategoryMask(true)
+                    .setOutputConfidenceMasks(false)
```

Comment:
Do we need to set both of these? I figured they each default to false.

Reply:
I just want to ensure that other developers can easily understand that there are two distinct modes.